src/docs/build-posix.dox

```diff
@@ -161,6 +161,35 @@ unless manually specified through the \c -DCC_OPTIMIZE_LEVEL configuration optio
 a different level of optimization:
 
 @code
-cmake -DCC_OPTIMIZE_LEVEL=-01 -G Ninja ../.
+cmake -DCC_OPTIMIZE_LEVEL=-O1 -G Ninja ../.
+@endcode
+
+@section posix_ctest Running WiredTiger C/C++ Tests
+
+The WiredTiger CMake build makes available a suite of C/C++ based tests.
+To run all the tests available, ensure you're in the build directory and execute:
+
+@code
+ctest -j$(nproc)
+@endcode
+
+To run with verbose output:
+
+@code
+ctest -j$(nproc) -VV
+@endcode
+
+To run a specific test:
+
+@code
+# Note: -R specifies a regex, where any matching test will be run
+ctest -R <test_name> -j$(nproc)
+@endcode
+
+To run our smoke tests, you can use the \c check alias:
+
+@code
+# Note: -R specifies a regex, where any matching test will be run
+ctest -L check -j$(nproc)
```

Comment:
There's some variance in how things are written that I think might be confusing. I'd suggest:
* making "-j $(nproc)" the first argument in all cases,
* removing the angle brackets in the -R case (unless they have to be there??), 
* removing the comment on line 192, it's a cut-and-paste error

I'd also maybe group the "run a smoke test" with the "run all tests" command, that is, the story this paragraph tells is: "here are the two things you can pick from -- all tests or the smoke tests, and if you want, you can also get verbose output, or run a specific test after you see what's failing."

Reply:
Thanks for the feedback! I've fixed up all those points :)